Shared/Extensions/URLExtensions.swift

```diff
@@ -155,7 +155,7 @@ extension URL {
      * E.g., https://m.foo.com/bar/baz?noo=abc#123  => foo
      **/
     public var hostSLD: String {
-        guard let publicSuffix = self.publicSuffix, let baseDomain = self.baseDomain else {
+        guard let publicSuffix = self.publicSuffix, let baseDomain = self.normalizedHost else {
```

Comment:
Methinks this function is no longer returning SLD-only. I would think our test cases would fail with this change.
Or, to make this change do:
`public var hostSLDAndNormalizedSubdomain` (sigh, the verbose naming convention for stripping www|m subdomain is PITA, I am at a loss for another naming).
and change `let baseDomain` in this function to `normalizedHost`.